shared/src/proxies/reports/getCustomCaseReportProxy.ts

```diff
@@ -2,10 +2,10 @@ import {
   GetCustomCaseReportRequest,
   GetCustomCaseReportResponse,
 } from '@web-api/business/useCases/caseInventoryReport/getCustomCaseReportInteractor';
+import { applicationContext } from '@web-client/applicationContext';
```

Comment:
You added an import for `applicationContext` but removed the reference to `applicationContext` below?

Reply:
Yeah an interesting case. So if we are to adopt this strategy we have to do it iteratively, and nearly function in the entire app requires appContext, so how do you remove appContext from a parent function without having to rewrite every function in the application?

For functions you do not want to update yet, import applicationContext instead of DI it to allow iterative changing.